src/main/java/com/blockout/Render.java

```diff
@@ -54,18 +54,18 @@ public static void drawOutlineRect(int x1, int y1, int x2, int y2, float lineWid
         float b = (float) ((color & COLOR_MASK) / COLOR_DIVISOR);
 
         Tessellator tessellator = Tessellator.getInstance();
-        WorldRenderer worldrenderer = tessellator.getWorldRenderer();
+        VertexBuffer VertexBuffer = tessellator.getBuffer();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)